src/main/java/datawave/microservice/authorization/postgres/PostgresDULProperties.java

```diff
@@ -0,0 +1,111 @@
+package datawave.microservice.authorization.postgres;
+
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+@EnableConfigurationProperties(PostgresDULProperties.class)
+@ConfigurationProperties(prefix = "postgres.config")
+public class PostgresDULProperties {
+    private String dbAddr;
+    private String dbName;
+    private String dbUser;
+    private String dbPassword;
+    private boolean rolesEnabled = true;
+    private String serverDnRegex;
+    private String cnRegex = "(cn=)?(?<sn>[a-zA-Z/.1-9]*),?";
```

Comment:
Couple of things about this regex. We're trying to match something like `[SOMETHING],cn=test.user.3,dn=[SOMETHING]` and then use a named group to extract `test.user.3`, right? 
You have a `?` after the cn meaning that it doesn't have to be there. So in the example I outlined above it would also match the `<sn>` group to the second `[SOMETHING]`. 
Then you have `/` and `.` in the square brackets. I'm guessing you're trying to escape the `.` but you don't need to and you are using the wrong slash if you were. As is you're telling it to match the `/` and `.` characters. We do need it to match `.` so that part is correct but I don't know if we need `/`. Also, do we know if hyphenated names are a thing with them?
Lastly you do `1-9`, this should probably be `0-9` otherwise it'll fail to catch things like `10`

Reply:
1) its a good thing the regex can be changed via config file. :)
2) basically, Im trying to match 3 things. 
opt A: SOMETHING, cn=test.user, DN=SOMETHING, SOMETHING
opt B: cn=test.user, SOMETHING, MORE
opt C: test.user

All those `test.user` could have numbers at the end. 

If we also want it CaC friendly, then really it should handle three groups with dots between them. Regex is complicated...